### PR TITLE
Add minimal info and main contacts to satellite page

### PR DESCRIPTION
**Pull request by leewujung**
Thinking it's good to have minimal and list main contacts for now, when details are being flushed out.

This builds on top of changes in #135. (in retrospect probably not a smart choice... )

### CODE REVIEW

**Comment by abkfenris** [comment]
Maybe I should just make pages for them too, even if they are really just stubs?

File: ohw22/satellites.md [hunk 1]
```diff
@@ -37,27 +37,40 @@ These two sub-programs are coordinated in their planning and coincides in time.
...
 ## OHW-Español
 
 - Time zone: UTC-4
 - Format: virtual + small in-person gatherings
-- Location: Lima, Peru and Rocha, Uruguay
-- In Spanish / bilingual
+- Locations: Lima, Peru and Rocha, Uruguay
+- Main contact: Emilio Mayorga (emiliom@uw.edu).
+
+Details of the event are still being flushed out, including the exact venues at the two locations. Check back here soon for more info!
```
Comment (emiliom) [inline ohw22/satellites.md:73]: I'm working on this right now. I'm adding a new satellite page, `espanol/index.md`

File: ohw22/satellites.md [hunk 2]
```diff
@@ -37,27 +37,40 @@ These two sub-programs are coordinated in their planning and coincides in time.
...
 - Time zone: UTC+10
 - Format: informal in-person gathering
-- Location: TBD
-- Limited logistic support available for regional participants needing to travel. See PAGE for detail.
+- Location: TBD, likely near Brisbane
+- Main contact: Nick Mortimer (Nick.Mortimer@csiro.au)
+
+We will have limited support available for regional participants needing to travel.
+
+Details of the event are still being flushed out, including the exact location. Check back here soon for more info!
```
Comment (emiliom) [inline ohw22/satellites.md:54]:
```suggestion
Details of the event are still being fleshed out, including the exact location. Check back here soon for more info!
```

File: ohw22/brazil/index.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# UFSC Satellite
+
+This year we are having two parallel East Coast events in Maine.
+One event will be held at Bigelow Labs in Boothbay, and the other at Gulf of Maine Research Institute in Portland
```
Comment (emiliom) [inline ohw22/brazil/index.md:4]:
```suggestion
```

File: ohw22/satellites.md (hunk 2 above)
Comment (leewujung) [reply ohw22/satellites.md]: Ahahahah, thanks for catching this!!

File: ohw22/brazil/index.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# UFSC Satellite
```
Comment (emiliom) [inline ohw22/brazil/index.md:1]:
```suggestion
# UFSC Satellite (Florianópolis, Brazil)
```

File: ohw22/satellites.md [hunk 5]
```diff
@@ -37,27 +37,40 @@ These two sub-programs are coordinated in their planning and coincides in time.
 - Time zone: UTC-7 (US/Pacific)
 - Format: informal in-person gathering
 - Location: San Diego (Scripps Institution of Oceanography), California, USA
+- Main contact: Steve Diggs (sdiggs@ucsd.edu)
+
+Details of the event are still being flushed out. Check back here soon for more info!
```
Comment (emiliom) [inline ohw22/satellites.md:42]:
```suggestion
Details of the event are still being fleshed out. Check back here soon for more info!
```

File: ohw22/satellites.md [hunk 6]
```diff
@@ -37,27 +37,40 @@ These two sub-programs are coordinated in their planning and coincides in time.
...
+We will have limited support available for regional participants needing to travel.
+
+Details of the event are still being fleshed out, including the exact location. Check back here soon for more info!
+
+
 
 ## Brazil
 
 - Time zone: UTC-3
 - Format: virtual gathering
```
Comment (emiliom) [inline ohw22/satellites.md:61]:
```suggestion
- Format: in-person gathering + virtual
```

**Comment by emiliom** [comment]
I made a bunch of minor tweaks (some not so minor, due to stray text that should've been deleted :laughing: ). Everything looks good, so I'll merge.

**Pull request merged**